tests/test_acl.py

```diff
@@ -492,3 +492,7 @@ def test_AclActionValidation(self, dvs, dvs_acl):
                 dvs.runcmd("supervisorctl restart syncd")
                 dvs.stop_swss()
                 dvs.start_swss()
+# Add Dummy always-pass test at end as workaroud
+# for issue when Flaky fail on final test it invokes module tear-down before retrying
+def test_nonflaky_dummy():
```

Comment:
Just double-checking, this works when added outside of the classes, right? We won't have this syslog issue if, say, the last test case in `TestAcl` fails?

Reply:
@daall : Outside of class works fine. Have verified it tes_acl.py also. 
Module teardown happens after executing dummy test case